### DISCUSSION TITLE
Unable to get good performance

### DISCUSSION THREAD

**Discussion by HamOntG** [vladmandic/sdnext#3339] [Q&A] [answered]
Hey, hoping I can get some help. I've followed all the recommended installation steps and even reinstalled Windows to make sure I have a clean slate, but I still have poor performance:
![image](https://github.com/user-attachments/assets/e429caf3-fa8a-4d09-a038-2c7f5367f090)
![image](https://github.com/user-attachments/assets/f2a911c6-612d-49f0-b26f-f3d766567174)
RTX 4090
Intel 13700K
64GB DDR5 RAM
Hoping this can help you point me in the right direction.

thanks!

**Comment by brknsoul** [comment, marked as answer]
If that's your benchmark, then that benchmark is pretty good for running an SDXL model.

**Reply by HamOntG** [reply]
Many thanks for the reply. I wasn't aware that benchmark performance depended on the model. I'll do some more testing.

Cheers!
